kornia/augmentation/container/augment.py

```diff
@@ -1,18 +1,20 @@
 import warnings
 from typing import Any, Dict, List, Optional, Tuple, Union, cast
 
-from kornia.augmentation import AugmentationBase3D, RigidAffineAugmentationBase2D, RigidAffineAugmentationBase3D
+import kornia.augmentation as K
```

Comment:
I’d would explicitly import what we need

Reply:
some circular dependencies/importing order issues here. This is easier to solve.